examples/with-validation/updatePost.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"strconv"
+	"withvalidation/types"
+)
+
+func UpdatePost(c *gin.Context) {
```

Comment:
Might be worth including go validator as a step here on the bindings

Reply:
What do you mean here?

Reply:
You mentioned using go validator in the PR, would be nice to use in an example - but I'm not sure if gin natively runs the validation step under its unmarshalling

Reply:
Yes gin natively runs the validation after is has unmarshaled the json data into the struct.

Reply:
Ok ignore me then.